Fix various local env issues and update the README

sarayourfriend opened the pull request.

sarayourfriend committed:
Fix timezone

sarayourfriend committed:
Move envs to correct location

sarayourfriend committed:
Update README with better instructions

sarayourfriend committed:
Convert README to markdown

sarayourfriend committed:
Add CI

sarayourfriend commented:
Not sure why the CI isn't running 🤔 Probably some kind of yaml syntax error?

sarayourfriend commented:
Never mind, I had duplicate keys. Figured it out 😅

AetherUnbound commented on .github/workflows/pull_request.yml:
```diff
@@ -0,0 +1,22 @@
+name: Test
```
I might rename this `Lint and Test` - also, would it be okay to include `black` and `flake8` here as well? Those are static checks that don't need to run within the docker-compose setup, they can run on the code directly. (I think `mypy` can also run staticly w/o an environment but don't quote me on that)

sarayourfriend replied:
Yeah actually we could just move those to a pre-commit hook 😀

sarayourfriend replied:
> I think mypy can also run staticly w/o an environment but don't quote me on that

I tried to get it to run but inside pre-commit it required too many dependencies and then environment variables that didn't exist. It seemed to make more sense to just run it inside the container where it already works.

AetherUnbound replied:
Gah, okay I'm remembering now that one of my work project runs mypy inside the container too 😕 ah well, thanks for trying that and adding the pre-commit file!

mpuckett159 committed:
Added some clarity for README instructions

Got a little confused with the order of operations described in the README so I updated it to add some clarification around the need to attempt logging in to the web app after you run the createsuperuser command, as well as an alternative option for viewing the logs of the django service a little more easily.

sarayourfriend committed:
Run pre-commit

AetherUnbound reviewed (approved):
🟢
